tests/testthat/test-if_else_match_braces_linter.R

```diff
@@ -0,0 +1,64 @@
+test_that("if_else_match_braces_linter skips allowed usages", {
+  expect_lint("if (TRUE) 1 else 2", NULL, if_else_match_braces_linter())
+  expect_lint("if (TRUE) 1", NULL, if_else_match_braces_linter())
+
+  lines_brace <- trim_some("
+    if (TRUE) {
+      1
+    } else {
+      2
+    }
+  ")
+  expect_lint(lines_brace, NULL, if_else_match_braces_linter())
```

Comment:
Just curious, why extract the `trim_some()` into a variable?

Reply:
we do so internally with a preference erring on the side of assigning variables, especially in tests (where if the test fails, we can look at the intermediate output more easily).

i might not write it the same from scratch, but avoiding extra diffs from our internal source where possible